pkg/util/create_or_update_test.go

```diff
@@ -338,6 +360,23 @@ func (t *createOrUpdateTestDriver) testUpdate(doUpdate func(util.OperationResult
 			tests.EnsureNoActionsForResource(t.testingFake, "pods/status", "update")
 		})
 
+		Context("and GenerateName is set", func() {
```

Comment:
I don’t think the Kubernetes APIs support `GenerateName` on update operations.

Reply:
See above comment re: delete.